### PR TITLE
Pokemon Page improvements

### PR DESCRIPTION
**Pull request by 123FLO321**
## Description
Added Pokemon Tree to Pokemon page
Added Raid count to Pokemon page
The Pokemon Tree is fully dynamic based on max pokemon id and also shows evolution items.

## How Has This Been Tested?
On local instance and working perfectly

## Screenshots (if appropriate):

Raid Count:
sc 1

Tree:
sc 2

sc 3

## Types of changes
- [X] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by michikrug** [comment]
I think the raid stuff will not work properly, because RM only stores the current raid for each gym.
So if a gym gets a new raid the old data is gone.

The evolution tree will maybe work also without a new file, because the candy requirements are stored in the pokedex file already.

**Comment by 123FLO321** [comment]
@michikrug 
1. True. Will add a new cron to fix that.
2. That would be possible, but I think it's cleaner this way. (and potentially faster)

**Comment by BlackRedStar1988** [comment]
I try to test it and the tree works without problems at all. And the raid too but i must wait to tomorrow for new raids. :D Nice Work.

**Comment by BlackRedStar1988** [comment]
Result runs without Problems at me.

**Comment by Obihoernchen** [comment]
looks really nice! +1 and I'll merge

**Comment by LordJavi** [comment]
Tested and works fine!

**Comment by Obihoernchen** [comment]
Please rebase :)

**Comment by 123FLO321** [comment]
done @Obihoernchen

File: pages/pokemon.page.php [hunk 1]
```diff
@@ -252,11 +275,76 @@
...
+					foreach ($data as $obj) {
+						$obj_id = $obj->id;
+						$tree_pokemon = $pokemons->pokemon->$obj_id;
+						$link = "/pokemon/".$obj_id;
 
+						if ($i % 2 == 0) { ?>
+
+							<div>
+								<a href="<?= $link ?>"><img src="<?= $tree_pokemon->img ?>" alt="<?= $tree_pokemon->name ?>" class="img"></a>
+								<p class="pkmn-name"><a href="<?= $link ?>">#<?= sprintf('%03d<br>%s', $tree_pokemon->id, $tree_pokemon->name) ?></a></p>
```
Comment (Obihoernchen) [inline pages/pokemon.page.php:308]: $link does not work if Worldopole is installed in a subdirecotry

**Review by Obihoernchen — CHANGES_REQUESTED** [review]

**Comment by javisabalete** [comment]
Works fine! Great job!

**Review by Obihoernchen — APPROVED** [review]

**Pull request merged**